robocluster/net.py

```diff
@@ -63,7 +64,7 @@ def bind(self):
         self._socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
 
         ip, port = self._address
-        if self.is_multicast:
+        if self.is_multicast and os.name == 'nt':
```

Comment:
Actually, could you add a quick comment as to why this is needed? It's easy to forget what exactly the problem was.

Reply:
The setting the ip to `''` that is.